DigitalLearningSolutions.Data/Models/User/AdminUser.cs

```diff
@@ -41,6 +41,8 @@ public class AdminUser : User
         public bool IsCmsAdministrator => ImportOnly && IsContentManager;
         public bool IsCmsManager => IsContentManager && !ImportOnly;
 
+        public bool Active { get; set; }

```

Comment:
We already have a property called `Active` it's on the `User` class that `AdminUser` extends.

Reply:
It seems Active property was added to User base class at the same time as it was added to AdminUser. I have removed it now.